### PR TITLE
changes for readme for manual end to end

### PR DESCRIPTION
**Pull request by katterianjali**
Readme is updated for future implementation with more steps.

### CODE REVIEW

File: demo/psa/manual-end-to-end.md [hunk 1]
```diff
@@ -5,13 +5,58 @@ end to end PSA Demonstration.
 
 ## Preconditions
 
-* One need to install `jq` and `curl`.
+* One need to install `GO`, `jq`,`sqilte3`,`tmux` and `curl`.
+
+Installing Go, use the below mentioned path to install go in your system.
+
+`https://go.dev/doc/install`
+
```
Comment (thomas-fossati) [inline demo/psa/manual-end-to-end.md:13]: These look like instructions that are targeted at Debian/Ubuntu systems.  We need to make sure this is made clear to the reader (e.g., by collating them under a `### Install on Ubuntu` sub-section.)

File: demo/psa/manual-end-to-end.md [hunk 2]
```diff
@@ -5,13 +5,58 @@ end to end PSA Demonstration.
...
+installing jq.
+```sh
+sudo apt update||sudo apt install jq||jq --version
```
Comment (thomas-fossati) [inline demo/psa/manual-end-to-end.md:16]: Those `||` should be `&&` instead?

File: demo/psa/manual-end-to-end.md [hunk 3]
```diff
@@ -5,13 +5,58 @@ end to end PSA Demonstration.
...
+sudo apt update
+sudo apt install sqlite3 
+sqlite3 --version
+```
+You will get an output like this:
+`3.31.1 2020-01-27 19:55:54 3bfa9cc97da10598521b342961df8f5f68c7388fa117345eeb516eaa837balt1`
+
+Installing Curl.
+```sh
+sudo apt-get install autoconf automake libtool curl make g++ unzip
```
Comment (thomas-fossati) [inline demo/psa/manual-end-to-end.md:30]: not an Ubuntu user, but I am a bit puzzled by the huge list of co-packages here.

I'd have thought a simple `sudo apt install curl` would suffice?

File: demo/psa/manual-end-to-end.md [hunk 4]
```diff
@@ -5,13 +5,58 @@ end to end PSA Demonstration.
...
+```
+
+Installing tmux.
+```sh
+sudo pacman -S tmux --noconfirm
```
Comment (thomas-fossati) [inline demo/psa/manual-end-to-end.md:35]: Not sure why pacman is needed?

Isn't `sudo apt-get install tmux` enough?

File: demo/psa/manual-end-to-end.md [hunk 5]
```diff
@@ -5,13 +5,58 @@ end to end PSA Demonstration.
...
 1. protoc-gen-go with version v1.26
+From  `https://github.com/protocolbuffers/protobuf/releases` download the `protobuf-all-[VERSION].tar.gz`.
+Extract the contents and change in the directory
+
+```sh
+./configure
+make
+make check
+sudo make install
+```
```
Comment (thomas-fossati) [inline demo/psa/manual-end-to-end.md:50]: I think there is a pre-built package `protobuf-compiler` that one can `sudo apt install`

File: demo/psa/manual-end-to-end.md [hunk 6]
```diff
@@ -5,7 +5,63 @@ end to end PSA Demonstration.
...
+Installing Go, use the below mentioned path to install go in your system.
+
+`https://go.dev/doc/install`
+
+installing jq.
+```sh
+sudo apt update && sudo apt install jq && jq --version
+```
+
+Installing sqlite3.
```
Comment (thomas-fossati) [inline demo/psa/manual-end-to-end.md:20]:
```suggestion
Installing sqlite3:
```

File: demo/psa/manual-end-to-end.md [hunk 7]
```diff
@@ -5,7 +5,63 @@ end to end PSA Demonstration.
...
+```sh
+sudo apt update
+sudo apt install sqlite3 
+sqlite3 --version
+```
+You will get an output like this:
+`3.31.1 2020-01-27 19:55:54 3bfa9cc97da10598521b342961df8f5f68c7388fa117345eeb516eaa837balt1`
+
+Installing Curl.
```
Comment (thomas-fossati) [inline demo/psa/manual-end-to-end.md:29]:
```suggestion
Installing curl:
```

File: demo/psa/manual-end-to-end.md [hunk 8]
```diff
@@ -5,7 +5,63 @@ end to end PSA Demonstration.
...
+```sh
+sudo apt install curl
+```
+
+Installing tmux.
```
Comment (thomas-fossati) [inline demo/psa/manual-end-to-end.md:34]:
```suggestion
Installing tmux:
```

File: demo/psa/manual-end-to-end.md (hunk 5 above)
Comment (thomas-fossati) [reply demo/psa/manual-end-to-end.md:50]: Have you checked my suggestion above?

**Review by thomas-fossati — COMMENTED** [review]
A few typos corrections inlined.

(Also a question about the `protoc` package.)

**Comment by thomas-fossati** [comment]
Ah, another small nit: it should be possible to run `sudo apt update` once for all the packages, right?

File: demo/psa/manual-end-to-end.md (hunk 5 above)
Comment (katterianjali) [reply demo/psa/manual-end-to-end.md:50]: Hi, The given steps worked for me not the pre built package. Should I change it to the pre built ?

Comment (thomas-fossati) [reply demo/psa/manual-end-to-end.md:50]: > The given steps worked for me not the pre built package.

how did it fail?

> Should I change it to the pre built ?

only if it works :-)

Comment (katterianjali) [reply demo/psa/manual-end-to-end.md:50]: these given commands works for me.

**Review by thomas-fossati — APPROVED** [review]
LGTM, thanks!

**Pull request merged**